src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+
+public class Storage {
+    private ArrayList<Task> store;
+    private PrintWriter writer;
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void storeData(TaskList t) {
+        store = t.getStore();
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < store.size(); i++) {
+            Task curr = store.get(i);
+            switch (curr.getType()) {
+                case TODO:
```

Comment:
Within this switch block, everything should be indented 4 spaces to the left (according to the coding standards). This being said, good job on adhering to the coding standards in general!